src/pages/jobs.js

```diff
@@ -0,0 +1,160 @@
+import { useState } from 'react'
+import Form from '../components/Jobs/Form/Form'
+import Card from '../components/Jobs/Card/Card'
+import Loader from '../components/Jobs/Loader/Loader'
+import Video from '../components/Jobs/Video/Video'
+import Pagination from '../components/Pagination/Pagination'
+import PageHeader from '../components/PageHeader'
+import { StaticImage } from 'gatsby-plugin-image'
+import SEO from '../components/SEO'
+
+function Jobs() {
+  const [jobData, setJobData] = useState(false)
+  const [formSubmitEvent, setFormSubmitEvent] = useState(false)
+  const [formSubmitted, setFormSubmitted] = useState(false)
```

Comment:
I would change to `isFormSubmitted` or `hasFormSubmitted` to indicate a boolean

Reply:
done, it helps distinguish it from formSubmitEvent a little as well.